lib/chai/core/assertions.js

```diff
@@ -679,14 +684,15 @@ module.exports = function (chai, _) {
   /**
    * ### .least(value)
    *
-   * Asserts that the target is greater than or equal to `value`.
+   * Asserts that the number target is greater than or equal to `value`.
    *
    *     expect(10).to.be.at.least(10);
    *
    * Can also be used in conjunction with `length` to
    * assert a minimum length. The benefit being a
    * more informative error message than if the length
-   * was supplied directly.
+   * was supplied directly. In this case the target must
+   * have a length property.
    *
    *     expect('foo').to.have.length.of.at.least(2);
    *     expect([ 1, 2, 3 ]).to.have.length.of.at.least(3);
```

Comment:
Another side note here:

Currently we can't do:
`expect([ 1, 2, 3 ]).to.have.a.length.of.at.least(3);`
Because the `a` word returns a function which has a property called `.length` and there is no way we can remove that in some environments, but we could use a `Proxy` in environments that do support it to override what happens when trying to access `length` on the function returned by `addChainableBehavior`, what do you think? Is this possible? I didn't dig deep on this one, but I think it might be a valid strategy.